packages/cli/src/utils/config.ts

```diff
@@ -1,24 +1,149 @@
-// TODO unfinished placeholder
+import { findUpSync } from "find-up";
+import path from "path";
+import { SchemaType } from "@latticexyz/schema-type";
+
+// TODO require may or may not be needed, finish exploring the various configurations
+import { createRequire } from "module";
```

Comment:
What might it be needed for? Do you want to do this exploration in this PR or as a follow up?

Reply:
(this is now in `storeConfig.ts`)

Reply:
yea I just renamed it. I might do `.cts` and `.ts` support later and might need requires (or might not).
Right now I'm also doing some config reorg so it's easier for us to work on it simultaneously (this require stuff will be segregated into a low level file you wouldn't need to change).
Should take ~hour or less, can merge then.